releases/4.0.0-beta.2.md

```diff
@@ -0,0 +1,54 @@
+---
+title: TiDB 4.0.0-beta.2 Release Notes
+category: Releases
+---
+
+# TiDB 4.0.0-beta.2 Release Notes
+
+发版日期：2020 年 03 月 13 日
+
+TiDB 版本：4.0.0-beta.2
+
+TiDB Ansible 版本：4.0.0-beta.2
+
+## TiDB 4.0.0-beta.2 兼容性变化
+
++ Tools
+    - TiDB Binlog
+        - 对 Drainer 的配置项 `disable-dispatch` 和 `disable-causality` 做了兼容 [#915](https://github.com/pingcap/tidb-binlog/pull/915)
```

Comment:
Please check the updated version in the Google Docs and update this note accordingly.